packages/cmd/src/help/WebHelpGenerator.ts

```diff
@@ -320,17 +315,13 @@ export class WebHelpGenerator {
             { commandDefinition: definition, fullCommandTree: this.mFullCommandTree, skipTextWrap: true });
 
         let markdownContent = helpGen.buildHelp() + "\n";
-        markdownContent = markdownContent.replace(/</g, "&lt;").replace(/>/g, "&gt;");
         if (definition.type === "group") {
             // this is disabled for the CLIReadme.md but we want to show children here
             // so we'll call the help generator's children summary function even though
             // it's usually skipped when producing markdown
             markdownContent += this.buildChildrenSummaryTables(helpGen, rootCommandName + "_" + fullCommandName);
         }
 
-        // Prevent line breaks from being lost during Markdown to HTML conversion
-        markdownContent = markdownContent.replace(/^(\s+Default value:.+$)(\s+Allowed values:.+$)/gm, "$1\n$2");
```

Comment:
This "hack" is no longer needed, thanks to https://github.com/zowe/zowe-cli/pull/1275